kraken-std/src/kraken/std/python/tasks/black_task.py

```diff
@@ -25,6 +25,8 @@ class BlackTask(EnvironmentAwareDispatchTask):
     def get_execute_command(self) -> list[str]:
         command = ["black", str(self.settings.source_directory)]
         command += self.settings.get_tests_directory_as_args()
+        if self.settings.lint_enforced_directories is not None:
+            command += [str(directory) for directory in self.settings.lint_enforced_directories]
```

Comment:
If `lint_enforced_directories` is not nullable, then this (and other tasks) can be simplified as

```suggestion
        command += [str(directory) for directory in self.settings.lint_enforced_directories]
```